.teamcity/src/main/kotlin/common/Jvm.kt

```diff
@@ -30,5 +30,5 @@ object BuildToolBuildJvm : Jvm {
     override val version: JvmVersion
         get() = JvmVersion.java11
     override val vendor: JvmVendor
-        get() = JvmVendor.adoptiumopenjdk
+        get() = JvmVendor.openjdk
```

Comment:
Does that mean that Temurin returns `OpenJdk`?

Reply:
Because now `linux.javaX.openjdk.64bit` points to temurin now, so this change is fine.